pages/development/theming.mdx

```diff
@@ -1,21 +1,270 @@
 
-We’re soon introducing a more advanced theming system, allowing you to:
+```tsx
+{
+  charts: {
+    colors: ["#c5fae2", "#7ef2be", "#2ce391", "#00cc70", "#009954", "#008348"],
+  },
+  font: {
+    family: "Inter",
+  },
+}
+```
 
-- Create Multiple Themes (e.g., “Dark Mode”, “Corporate Brand”, “Holiday Special”, "Customer A Colors").
+That would apply the Inter font across the entire site and change all of the chart colors. We handle applying any javascript color values (which are mostly used for charts) and we also generate CSS Variables from the theme file which are then used by our controls, inputs, etc.
 
-- Granular Control over each element (beyond fonts and colors).
+### Single Theme
 
-- Switch Themes Dynamically via [clientContext](/development/client-context), letting you pass in a theme name or user preference.
+If you just want to have a single theme, it’s a very straightforward process! You’ll need to find `embeddable.theme.ts` at the root level of the Boilerplate Repo. Here you’ll see that we’ve added a couple of values to get you started, but also that we’re importing `Theme` from the Vanilla Components, which will give you full type definitions for the entire theme object if you want to dig in! The file looks like this:
 
-With this enhancement, you’ll be able to swap themes on the fly—perfect for supporting dark mode or user-specific branding without redeploying or rebuilding. Stay tuned!
\ No newline at end of file
+```tsx
+import { Theme, defineTheme } from 'vanilla-components';
+
+const themeProvider = (clientContext: any, parentTheme: Theme): Theme =>
+	defineTheme(parentTheme, {
+    brand: {
+      primary: '#008348',
+      secondary: '#BA9653',
+    },
+    charts: {
+      colors: [
+        '#c5fae2',
+        '#7ef2be',
+        '#2ce391',
+        '#00cc70',
+        '#009954',
+        '#008348',
+        '#006638',
+        '#004d2a',
+      ],
+    },
+    font: {
+      family: 'Inter',
+    },
+  });
+
+export default themeProvider;
+
+```
+
+Let’s break down what’s happening here. First, we’re bringing in the `Theme` type from Vanilla Components to help with type consistency, and the `defineTheme` function from the same repo, which we use to help us merge our changes into the default Vanilla Components theme. Our `themeProvider` function has `clientContext` (which we’re not using in this example, but see below!) and `parentTheme` parameters. Without that `parentTheme`, we wouldn’t get all of the values from Vanilla Components. You could still go without it, but you’d have to define every single property specified in `Theme` which, as of this writing, is about 130 lines of code. Should you want that level of granular control, just skip using `defineTheme` at all, and return your theme object directly.
+
+Once you’ve changed `embeddable.theme.ts`, save it, and you should be set. Your theme changes will show up across all of the Vanilla Components.
```

Comment:
@cwbuecheler maybe add a sentence to the end here, saying something like "Try it out using `embeddable:dev`" with a link to the relevant page in the docs